VersionControlUI/styles/item/HighlightOverlay/default

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE EnvisionVisualizationStyle>
+<style prototypes="item/DeclarativeItemBase/default">
```

Comment:
Oh, you called this style default. You should not do that. A default style already exists in VisualizationBase and it will be coppied to HighlightOverlay directory when you run `make install`. So basically here you should only define styles specifically for use of version control and you should name them accordingly.